sync.sh

```diff
@@ -1,37 +1,10 @@
 #!/bin/bash
 
-helpFunction()
```

Comment:
I think some help would still be nice, even if it doesn't take any args

Reply:
That adds quite a bit of complexity - we would have to maintain argparsing just for the sake of the help. I think I will leave this for now - but thanks for suggesting.